app/models/match.rb

```diff
@@ -3,4 +3,7 @@ class Match < ActiveRecord::Base
   has_many :games
   has_many :game_scores, through: :games
   has_many :users, through: :game_scores
+
+  has_one :ball_color
+  has_one :ball_quality  
```

Comment:
You can kill these lines, has_one is for associations between models, so you don't need them for these fields. You could have a validation maybe, but since they're not required probably not